test/e2e-common.sh

```diff
@@ -427,6 +445,8 @@ function knative_teardown() {
 	kubectl delete --ignore-not-found=true -f "${YAML}" || return 1
     done
   fi
+  # Delete the test namespace
+  kubectl delete namespace ${E2E_SYSTEM_NAMESPACE} --ignore-not-found=true
```

Comment:
I think we want to trap it, to properly clean up if the tests terminate abruptly

Reply:
+1